libraries/Microsoft.Bot.Builder.Dialogs/Choices/ChoiceFactory.cs

```diff
@@ -112,31 +115,34 @@ public static Activity List(IList<Choice> choices, string text = null, string sp
 
             // Format list of choices
             var connector = string.Empty;
-            var txt = text ?? string.Empty;
-            txt += "\n\n   ";
+            var txtBuilder = new StringBuilder()
```

Comment:
Again, I would actually suggest passing text to the constructor of the StringBuilder to let it initialize itself with that. (see comment above for more details)